core/SConscript.prodtest

```diff
@@ -43,7 +47,7 @@ if TREZOR_MODEL in ('1', 'R', 'T3B1'):
     FONT_NORMAL_UPPER=None
     FONT_BOLD_UPPER=None
     FONT_SUB=None
-elif TREZOR_MODEL in ('T',):
+elif TREZOR_MODEL in ('T','DISC2',):
```

Comment:
Pointless addition as the DISC2 build of prodtest is disabled above. We might actually want to enable it as i don't think there are reasons no to do that, but currently this won't work for other reason: because prodtest is using legacy drawing API, which doesn't work with 32bit colors DISC2 uses. 

We can keep though it as it doesn't really matter, apart from making impression that its supported while its not.

Reply:
I just tested it on DISC2 (together with npm1300 driver) and forgot to delete it. It's fixed here c1d9151b17c3b20b9f9b508148e44f8f98503eb9